x-pack/plugin/esql/qa/testFixtures/src/main/resources/categorize.csv-spec

```diff
@@ -374,6 +374,29 @@ COUNT():long | category:keyword
            7 | null
 ;
 
+on const null
+required_capability: categorize_v5
+
+FROM sample_data
+  | STATS COUNT(), SUM(event_duration) BY category=CATEGORIZE(null)
+  | SORT category
+;
+
+COUNT():long | SUM(event_duration):long | category:keyword
+           7 |                 23231327 |  null
+;
+
+on null row
+required_capability: categorize_v3
```

Comment:
This has to be `v5`, otherwise this test is not run.

Reply:
Fixed it. Just double checked, and the test fails as the capability doesn't exist, nice!